sailpoint.sgnl.yaml

```diff
@@ -0,0 +1,1575 @@
+        type: String
+      - name: reviewerValue
+        externalId: reviewer.value
+        type: String
+      - name: reviewerRef
+        externalId: reviewer.["$ref"]
+        type: String
+
+      # Meta
+      - name: resourceType
+        externalId: $.meta.resourceType
+        type: String
+      - name: created
+        externalId: $.meta.created
+        type: DateTime
+      - name: lastModified
+        externalId: $.meta.lastModified
+        type: DateTime
+      - name: version
+        externalId: $.meta.version
+        type: String
+      - name: location
+        externalId: $.meta.location
+        type: String
+
+  # TODO: Identify unique and index attributes
+  EntitlementClassifications:
+    parent: Entitlements
+    displayName: SailpointEntitlementClassifications
+    externalId: classifications
+    description: Entity representing a classification associated with a Sailpoint Identity IQ Entitlement.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: source
+        externalId: source
+        type: String
+      - name: effective
+        externalId: effective
+        type: Bool
+      - name: classificationName
+        externalId: classification.name
+        type: String
+      - name: classificationDisplayName
+        externalId: classification.displayName
+        type: String
+      - name: classificationOrigin
+        externalId: classification.origin
+        type: String
+      - name: classificationType
+        externalId: classification.type
+        type: String
+
+  EntitlementLocalizedDescriptions:
+    parent: Entitlements
+    displayName: SailpointEntitlementLocalizedDescriptions
+    externalId: descriptions
+    description: Entity representing the localized descriptions associated with a Sailpoint Identity IQ Entitlement.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: locale
+        externalId: locale
+        type: String
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+
+  EntitlementApplication:
+    parent: Entitlements
+    displayName: SailpointEntitlementApplications
+    externalId: application
+    description: Entity representing the application associated with a Sailpoint Identity IQ Entitlement.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: value
+        externalId: value
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: ref
+        externalId: $..["$ref"]
```

Comment:
Why are you using this notation here? This should be at the top level (e.g. just `$ref` should work)

https://developer.sailpoint.com/iiq/api/get-entitlements